docs/application/dotnet/guides/nui/ui-components.md

```diff
@@ -29,6 +29,7 @@ The following table lists the available UI components:
 | [TextLabel](./textlabel.md)          | A text label that renders a short text string. | `TextLabel`                |
 | [Toast](./nui-components/Toast.md) | A toast is used to provide simple messages. | `Toast`                |
 | [VideoView](./videoview.md)          | A video view that controls and displays video playback. | `VideoView`                 |
+| [Widget](./widget.md)                | A widget that create light Tizen .NET applications on the home screen | `Widget` |
```

Comment:
Suggest changing to:

A widget is used to create light Tizen .NET applications on the home screen.